test/test_utils/ecr.py

```diff
@@ -127,6 +174,33 @@ def get_ecr_image_scan_results(ecr_client, image_uri, minimum_vulnerability="HIG
     return scan_findings
 
 
+def get_all_ecr_enhanced_scan_findings(ecr_client, image_uri):
+    """
+    Get list of all vulnerabilities from ECR ENHANCED image scan results
+    :param ecr_client:
+    :param image_uri:
+    :return: list<dict> Scan results
+    """
+    scan_info_findings = get_all_ecr_image_scan_results_using_pagination(
+        ecr_client, image_uri, scan_info_finding_key="enhancedFindings"
+    )
+    scan_findings = []
+    for scan_info_finding in scan_info_findings:
+        scan_findings += [finding for finding in scan_info_finding]
```

Comment:
Is this for loop here to deal with the results of multiple function calls made to `describe_image_scan_findings` being directly put in a list and passed to the output of the `get_all_ecr_image_scan_results_using_pagination` function?

If so, why not just do this inside the `get_all_ecr_image_scan_results_using_pagination` function?

Reply:
Done